packages/lisk-api-client/test/errors.ts

```diff
@@ -13,16 +13,15 @@
  *
  */
 import { expect } from 'chai';
-import { APIError } from '../src/errors';
+import { APIError, APIErrorData } from '../src/errors';
 
 describe('api errors module', () => {
 	let apiError: APIError;
 	const defaultMessage = 'this is an error';
 	const defaultErrno = 401;
 
-	beforeEach(() => {
+	beforeEach(async () => {
```

Comment:
why `async`?

Reply:
I will put it back, we should at some point move everything to async